build.gradle

```diff
@@ -1,4 +1,11 @@
 buildscript {
+    ext {
+        kotlin_version = '1.8.20'
+    }
+    dependencies {
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.5.3"
```

Comment:
we don't use navigation library currently. So we don't need safe args. I'd suggest to revert changes in this file, it should work without them.

Reply:
removed line 6